tests/data_context/test_data_context.py

```diff
@@ -297,87 +298,89 @@ def test_list_datasources(data_context_parameterized_expectation_suite):
         },
     ]
 
-    # Make sure passwords are masked in password or url fields
-    data_context_parameterized_expectation_suite.add_datasource(
-        "postgres_source_with_password",
-        initialize=False,
-        module_name="great_expectations.datasource",
-        class_name="SqlAlchemyDatasource",
-        credentials={
-            "drivername": "postgresql",
-            "host": os.getenv("GE_TEST_LOCAL_DB_HOSTNAME", "localhost"),
-            "port": "65432",
-            "username": "username_str",
-            "password": "password_str",
-            "database": "database_str",
-        },
-    )
-
-    data_context_parameterized_expectation_suite.add_datasource(
-        "postgres_source_with_password_in_url",
-        initialize=False,
-        module_name="great_expectations.datasource",
-        class_name="SqlAlchemyDatasource",
-        credentials={
-            "url": "postgresql+psycopg2://username:password@host:65432/database",
-        },
-    )
+    if is_library_loadable(library_name="psycopg2"):
```

Comment:
The diff looks funky, but it's just moving that chunk of code under this conditional statement